starfish/starfish-core/src/query/mod.rs

```diff
@@ -400,7 +400,7 @@ impl Query {
             let stmt = sea_query::Query::select()
                 .column(NodeQueryIden::Name)
                 .expr_as(Expr::col(Alias::new(&weight_key)), NodeQueryIden::Weight)
-                .expr_as(Expr::val(Some(0_u64)), NodeQueryIden::Depth)
+                .expr_as(Expr::val(Option::<u32>::None), NodeQueryIden::Depth)
```

Comment:
Why this is changed from `Some(0_u64)` to `Option::<u32>::None`?
Isn't it be changed to `Some(0_u32)`?

Reply:
It was changed from `Some(0_u64)` to `Option::<u64>::None` at some point before (when I implemented counting depth), and then to `Option::<u32>::None` for PostgresQL support specifically.

As the depth is counted with Rust, this value is only a placeholder for querying the nodes. I suppose it doesn't matter practically whether `0` or `None` is used, but `None` seems to fit the meaning of placeholder better?